Src/Plugins/Security/PFXCryptographicKeyFileValidator.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    internal static class PFXCryptographicKeyFileValidator
```

Comment:
>PFXCryptographicKeyFileValidator [](start = 26, length = 32)

Please correct casing of this to PfxCryptographicKeyfileValidator. Note casing of 'Keyfile' this is one word. Change all rule id references as well. I notice that we aren't always capitalizing API as Api as well, we'll make a pass later.

One thing, you may have trouble changing the casing of this file name, I don't know. It's possible that your last casing will be retained when I sync (hopefully). To be sure, you can temporarily rename the file to foo.cs, push a commit, then rename to final casing (this works for sure).

[https://en.wikipedia.org/wiki/Keyfile](https://en.wikipedia.org/wiki/Keyfile)

Reply:
ok

---
In reply to: [554470314](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554470314) [](ancestors = 554470314)